services/api/src/admin_org.py

```diff
@@ -119,13 +140,26 @@ def get_allowed_selections():
     return obj
 
 
-def get_modify_org_data(org_name):
+@require_permission(
+    required_role=ORG_ROLE_LITERAL.USER,
+    resource_type=RESOURCE_TYPE.ORGANIZATION,
+)
+def get_modify_org_data_authorized(org_name: str, permission_result: PermissionResult):
     modify_org_obj = {}
     # Get list of all organizations or details of a singular organization
+    # Only requires "user" role to access this endpoint, as it is just counts
     if org_name == "all":
-        modify_org_obj["org_data"] = get_all_orgs()
+        if permission_result.user.user_info.super_user:
+            modify_org_obj["org_data"] = get_all_orgs(None)
+        else:
+            modify_org_obj["org_data"] = get_all_orgs(permission_result.qualified_orgs)
     else:
-        modify_org_obj["org_data"] = get_org_data(org_name)
+        # Only requires "user" role to access this endpoint, as it is just counts
+        isAdminInOrg = permission_result.user.user_info.super_user or check_role_above(
+            permission_result.user.user_info.organizations.get(org_name),
+            ORG_ROLE_LITERAL.ADMIN,
+        )
+        modify_org_obj["org_data"] = get_org_data(org_name, isAdminInOrg)
```

Comment:
polish(non-blocking): use standard naming convention
```suggestion
        is_admin_in_org = permission_result.user.user_info.super_user or check_role_above(
            permission_result.user.user_info.organizations.get(org_name),
            ORG_ROLE_LITERAL.ADMIN,
        )
        modify_org_obj["org_data"] = get_org_data(org_name, is_admin_in_org)
```

Reply:
done